server/src/main/java/org/elasticsearch/index/mapper/TypeParsers.java

```diff
@@ -341,6 +341,7 @@ private static IndexOptions nodeIndexOptionValue(final Object propNode) {
     }
 
     public static DateFormatter parseDateTimeFormatter(Object node) {
+        //todo pg debug this..
```

Comment:
this is used when using dynamic date formats setting. - so at index creation https://www.elastic.co/guide/en/elasticsearch/reference/current/dynamic-field-mapping.html